json_schemas/framework-agreement-details.json

```diff
@@ -15,6 +15,57 @@
       "minLength": 1,
       "type": "string"
     },
+    "frameworkRefDate": {
+      "minLength": 1,
+      "type": "string",
+      "patterm": "^[0-9-]+$"
+    },
+    "frameworkStartDate": {
+      "minLength": 1,
+      "type": "string",
+      "patterm": "^[0-9/]+$"
+    },
+    "frameworkEndDate": {
+      "minLength": 1,
+      "type": "string",
+      "patterm": "^[0-9/]+$"
```

Comment:
* There is no duplication here at all - we don't store these dates anywhere else in the database. (We do store the "applications closed" date, but that is none of the dates here.)

* The data here (i.e. in the frameworkAgreementDetails block) is *not* intended to be used anywhere at all on the frontend.  It is for use by the scripts that generate the signature page PDFs.

* The main requirement of those PDFs is that they *exactly* match the page in the full framework agreement document, so storing the *exact* strings that need to go in there makes sense.

* I'm not sure in what sense these are ambiguous at all?